packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -107,18 +109,18 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
         id={ this._id }
         role='checkbox'
         type='button'
-        className={ classNames.root }
+        className={ this._classNames.root }
         onClick={ this._onClick }
         onFocus={ this._onFocus }
         onBlur={ this._onBlur }
         aria-checked={ isChecked }
         aria-disabled={ disabled }
       >
-        <label className={ classNames.label } htmlFor={ this._id } >
-          <div className={ classNames.checkbox }>
-            <Icon iconName='CheckMark' className={ classNames.checkmark } />
+        <label className={ this._classNames.label } htmlFor={ this._id } >
+          <div className={ this._classNames.checkbox }>
+            <Icon iconName='CheckMark' className={ this._classNames.checkmark } />
           </div>
-          { label && <span className={ classNames.text }>{ label }</span> }
+          { label && onRenderLabel(this.props, this._onRenderLabel) }
```

Comment:
what if instead of a label I want to render an image (or whatever). That image won't be displayed unless you pass a string into the label prop. That seems an unexpected requirement to use the onRenderLabel